hedera-node/docker/Dockerfile

```diff
@@ -1,42 +1,48 @@
 ## required libraries and startup assets for a node with:
-##  * Configuration from /opt/hedera/services/config-mount; and,
-##  * Logs at /opt/hedera/services/output; and,
-##  * Saved states under /opt/hedera/services/output
-FROM ubuntu:20.04 AS base-runtime
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y dos2unix openssl libsodium23 bind9-host
-
-# JDK
-RUN apt-get install -y software-properties-common && \
-    add-apt-repository -y ppa:openjdk-r/ppa && \
-    apt-get install -y openjdk-17-jdk
-
-# Services runtime
-RUN mkdir -p /opt/hedera/services/data/lib
-RUN mkdir /opt/hedera/services/data/apps
-RUN mkdir /opt/hedera/services/data/config
-RUN mkdir /opt/hedera/services/data/saved
-RUN mkdir /opt/hedera/services/data/onboard
-RUN mkdir /opt/hedera/services/output
-RUN mkdir /opt/hedera/services/config-mount
-
-## Finishes by copying the Services JAR to the base runtime
-FROM base-runtime AS final-image
-
-WORKDIR /opt/hedera/services
-
-COPY start-services.sh /opt/hedera/services/start-services.sh
-
-COPY .env /opt/hedera/services
+##  * Configuration from /opt/hgcapp/services-hedera/HapiApp2.0/config-mount; and,
+##  * Logs at /opt/hgcapp/services-hedera/HapiApp2.0/output; and,
+##  * Saved states under /opt/hgcapp/services-hedera/HapiApp2.0/output
+ARG IMAGE_PREFIX="gcr.io/hedera-registry/"
+ARG IMAGE_TAG="0.43.0-alpha.6"
+
+FROM ${IMAGE_PREFIX}network-node-base:${IMAGE_TAG} AS final-image
+
+# Create services runtime locations and pre-touch logs
+RUN mkdir -p /opt/hgcapp/services-hedera/HapiApp2.0/data/lib \
+ && mkdir /opt/hgcapp/services-hedera/HapiApp2.0/data/apps \
+ && mkdir /opt/hgcapp/services-hedera/HapiApp2.0/data/config \
+ && mkdir /opt/hgcapp/services-hedera/HapiApp2.0/data/saved \
+ && mkdir /opt/hgcapp/services-hedera/HapiApp2.0/data/onboard \
+ && mkdir /opt/hgcapp/services-hedera/HapiApp2.0/output \
+ && mkdir /opt/hgcapp/services-hedera/HapiApp2.0/config-mount \
+ && mkdir -p "/opt/hgcapp/services-hedera/HapiApp2.0/output" \
+ && touch "/opt/hgcapp/services-hedera/HapiApp2.0/stdout.log" \
+
+WORKDIR /opt/hgcapp/services-hedera/HapiApp2.0
+
+# Add SDK components
+ADD sdk/data/apps/* /opt/hgcapp/services-hedera/HapiApp2.0/data/apps/
+ADD sdk/data/lib/* /opt/hgcapp/services-hedera/HapiApp2.0/data/lib/
+
+# Copy configuration and scripts
+COPY --from=services-data onboard/StartUpAccount.txt /opt/hgcapp/services-hedera/HapiApp2.0/data/onboard
+COPY start-services.sh /opt/hgcapp/services-hedera/HapiApp2.0/start-services.sh
+COPY .env /opt/hgcapp/services-hedera/HapiApp2.0/
+
+# Ensure proper file permissions, fill in version file, list libraries, and
+# ensure the start script has correct line endings
 RUN for PIECE in $(cat .env | head -1 | tr '=' ' '); do \
-  if [ "$IS_VERSION" = "true" ]; then echo $PIECE >> .VERSION ; else IS_VERSION=true; fi done
+  if [ "$IS_VERSION" = "true" ]; then echo $PIECE >> .VERSION ; else IS_VERSION=true; fi done \
```

Comment:
These lines are unnecessary, the `VERSION` file is only to satisfy devops requirements in the production deployments and have no merit being included in the docker container image.

Reply:
This is half done and not at all ready for review.  It's pushed to draft PR just to test in CI.